src/one_dragon_qt/widgets/install_card/code_install_card.py

```
@@ -31,6 +34,14 @@ def __init__(self, ctx: OneDragonEnvContext, parent=None):
 
         self.updated: bool = False  # 是否已经更新了
 
+    def fetch_latest_code(
+        self,
+        progress_callback: Callable[[float, str], None],
+    ) -> tuple[bool, str]:
+        """同步代码，并适配安装卡的布尔结果。"""
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)